src/pages/Account/Profile.tsx

```diff
@@ -1,16 +1,24 @@
 import React from "react";
 import { useForm } from "react-hook-form";
 import { useHistory } from "react-router-dom";
-import { updateUserProfile } from "./helpers";
 import "firebase/storage";
-import "./Account.scss";
-import ProfilePictureInput from "components/molecules/ProfilePictureInput";
-import { RouterLocation } from "types/RouterLocation";
-import { useUser } from "hooks/useUser";
+
 import { IS_BURN } from "secrets";
-import getQueryParameters from "utils/getQueryParameters";
+
 import { DEFAULT_VENUE, DISPLAY_NAME_MAX_CHAR_COUNT } from "settings";
+
 import { useVenueId } from "hooks/useVenueId";
+import { useUser } from "hooks/useUser";
+
+import getQueryParameters from "utils/getQueryParameters";
+
+import { RouterLocation } from "types/RouterLocation";
+
+import { updateUserProfile } from "./helpers";
+
+import ProfilePictureInput from "components/molecules/ProfilePictureInput";
```

Comment:
[nit] secrets, settings, types, utils, hooks, components, ./

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1409/commits/041df52e62699a3a610407e6d105970072a53ce2